osu.Game.Rulesets.Mania/Objects/Types/IHasColumn.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+namespace osu.Game.Rulesets.Mania.Objects.Types
+{
+    /// <summary>
+    /// A type of hit object which lies in one of a number of predetermined columns.
+    /// </summary>
+    public interface IHasColumn

```

Comment:
This is not for conversion to any other standard game modes. If other modes want to convert from mania they should reference the mania assembly explicitly and use this interface.